sdk/eventhub/azure-eventhub/azure/eventhub/_utils.py

```diff
@@ -74,9 +73,9 @@ def dst(self, dt):
 
 
 try:
-    from datetime import timezone  # pylint: disable=ungrouped-imports
+    from datetime import timezone
 
-    TZ_UTC = timezone.utc  # type: ignore
+    TZ_UTC: timezone = timezone.utc
 except ImportError:
     TZ_UTC = UTC()  # type: ignore
```

Comment:
this one as well for type :)

Reply:
you cannot remove the second type ignore here unless you cast UTC to timezone which did not make sense to me

Reply:
tbh I don't think we need this class anymore. `datetime.timezone` was added in py3.2, so the `ImportError` wont even be running. If we arent using the UTC class outside anywhere else we should get rid of it.